packages/scripts/src/create-webpack-configs.ts

```diff
@@ -291,14 +291,19 @@ export function createWebpackConfigForExternalFeature({
         },
         stats: 'errors-warnings',
     };
+    if (semverLessThan(webpack.version, '5.0.0')) {
```

Comment:
this was just added in the last rebase.
will make it compatible with both 4.0.0 and 5.0.0.